src/hooks/useReports.ts

```diff
@@ -23,7 +23,7 @@ const useReports = (ownerId?: string, gateway?: AoGateway) => {
   const { data: epochs } = useEpochs();
 
   const queryResults = useQuery({
-    queryKey: ['reports', ownerId],
+    queryKey: ['reports', ownerId, arIOReadSDK],
```

Comment:
- should add epochs to query key
- use enabled flag instead of throwing
- any gql query that queries with a set of ID's can be fetched with queryClient.fetchQuery since its a deterministic fetch with an inifinite stale time
- consider adding gateway to queryKey? not sure its useful but probably wouldnt hurt

Reply:
Added PE-7524 to look at this another time, want to move forward with this as-is since it's stable code and optimization here is better looked at in another story.